src/app/grid/grid-select/grid-select-sample.component.scss

```diff
@@ -0,0 +1,44 @@
+.grid__wrapper {
```

Comment:
Is any of this .scss file content needed, except for maybe some margins?

Reply:
Removed everything except the margin but change the scope to igx-grid.